### PR TITLE
rtmp-services: Update MyFreeCams

### PR DESCRIPTION
**Pull request by SCG82**
### Motivation and Context

* Temporarily removing the 'supported resolutions' constraint due to #4124.
* Increasing 'max video bitrate' to 10000 kbps.
### How Has This Been Tested?

Tested using a variety of bitrates/framerates/resolutions in macOS & windows.
### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -1868,17 +1868,12 @@
...
-                    "1920x1080",
-                    "1280x720",
-                    "960x540",
-                    "640x360"
-                ],
                 "max fps": 60,
-                "max video bitrate": 6000,
+                "max video bitrate": 10000,
                 "max audio bitrate": 192,
+                "bframes": 0,
```
Comment (dodgepong) [inline plugins/rtmp-services/data/services.json:1876]: `tune=zerolatency` should already be setting bframes to 0, so this seems redundant.

Comment (dodgepong) [reply plugins/rtmp-services/data/services.json:1876]: Actually, I suppose this setting would be used on hardware encoders? I not certain what OBS does with x264opts when you're not using x264.

Comment (SCG82) [reply plugins/rtmp-services/data/services.json:1876]: Yes, I added `bframes: 0` for hardware encoders (but haven't tested it yet)

Comment (jp9000) [reply plugins/rtmp-services/data/services.json:1876]: I don't really think it's necessary even for hardware encoders. Hardware encoders already have very low latency. I'd feel more comfortable without this.

Comment (SCG82) [reply plugins/rtmp-services/data/services.json]: removed

**Comment by SCG82** [comment]
rebased

**Pull request merged**